practices/backlog_cleanup.md

```diff
@@ -0,0 +1,59 @@
+---
+slug: backlog-cleaning
+stages:
+  - development
+short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
+tags:
+  - backlog cleaning
+  - agile
+  - scrum
+keywords:
+  - backlog
+  - cleaning
+  - grooming
+  - agile
+  - scrum
+author_github_username: cadamini
+---
+
+# Backlog Cleanup
+
+**TL;DR**
+
+A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
+
+## What Is Backlog Cleanup  
+
+Backlog Cleanup describes the process of managing issues in your product backlog. Your backlog may be full of old feature ideas.
+After some strategy meetings in the last months/quarters, your feature ideas may be out of scope, since the product strategy has shifted in a different direction.
```

Comment:
```suggestion
Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.
```